### DISCUSSION TITLE
HEIC support for Deepin Image Viewer

### DISCUSSION THREAD

**Discussion by SiamSami** [linuxdeepin/developer-center#6857] [Features Request & Ideas | 特性请求 & 头脑风暴]
It would be great if Deepin Image viewer could display HEIC format images

**Comment by BLumia** [comment]
Which distro are you using? If you are using Arch, try https://aur.archlinux.org/packages/qt5-heic-image-plugin

btw, we have a live chatroom if you're interested: [Telegram](https://t.me/deepin) [Matrix](https://matrix.to/#/#deepin:matrix.org)

**Reply by SiamSami** [reply]
I am using Deepin OS. I tried installing that package you mentioned but it still doesn't work

**Reply by BLumia** [reply]
Are you on v20 stable or v23 beta?

**Reply by SiamSami** [reply]
V20 stable
